.github/workflows/ci_linux.yml

```diff
@@ -7,44 +7,7 @@ on:
     branches: [ main ]
 
 jobs:
-  build:
-    runs-on: ubuntu-22.04
-
-    steps:
-      - uses: actions/checkout@v3
-
-      - name: Use Node.js 16
-        uses: actions/setup-node@v3
-        with:
-          node-version: 16
-
-      - name: Install Dependencies
-        run: yarn install
-        working-directory: Extension
-
-      - name: Compile Sources
-        run: yarn run compile
-        working-directory: Extension
-
-      - name: Run Linter
-        run: yarn run lint
-        working-directory: Extension
-
-      - name: Run unit tests
-        run: yarn test
-        working-directory: Extension
-
-#      # NOTE : We can't run the test that require the native binary files
-#      #        yet -- there will be an update soon that allows the tester to 
-#      #        acquire them on-the-fly
-#      - name: Run simple vscode unit tests
-#        uses: GabrielBB/xvfb-action@v1.6

```

Comment:
It's maybe worth keeping a note that running the tests that pop open VS Code need this xvfb action to allow for the UI to appear on mac/linux.  When these tests are re-enabled, you'll need to do it differently for Windows vs. mac/linux.

Reply:
Added conditionals for these tests that are platform dependent.